src/code/entities/06_zol.asm

```diff
@@ -150,44 +171,45 @@ jr_006_7CB7:
 .jr_7CD4
     ldh  a, [hActiveEntityState]                  ; $7CD4: $F0 $F0
     cp   $04                                      ; $7CD6: $FE $04
-    jr   z, .jr_7CE5                              ; $7CD8: $28 $0B
+    jr   z, .skipCollision                        ; $7CD8: $28 $0B
 
     ld   hl, wEntitiesPrivateCountdown3Table      ; $7CDA: $21 $80 $C4
     add  hl, bc                                   ; $7CDD: $09
     ld   a, [hl]                                  ; $7CDE: $7E
     and  a                                        ; $7CDF: $A7
-    jr   nz, .jr_7CE5                             ; $7CE0: $20 $03
+    jr   nz, .skipCollision                       ; $7CE0: $20 $03
 
     call DefaultEnemyDamageCollisionHandler_trampoline ; $7CE2: $CD $39 $3B
 
-.jr_7CE5
+.skipCollision
     ldh  a, [hActiveEntityState]                  ; $7CE5: $F0 $F0
     JP_TABLE                                      ; $7CE7
-._00 dw ZolState0Handler
-._01 dw ZolState1Handler
-._02 dw ZolState2Handler
-._03 dw ZolState3Handler
-._04 dw ZolState4Handler
-
-ZolState1Handler::
+._00 dw ZolGelState0Handler
+._01 dw ZolGelState1Handler
+._02 dw ZolGelState2Handler
+._03 dw ZolGelState3Handler
+._04 dw GelState4Handler
+
+; Inching (up)
+ZolGelState1Handler::
```

Comment:
In general, feel free to rename these handlers when their purpose has been identified.

In that case, `ZolGelInchingHandler` would be perfect.

(But the comments are already great; it can be for another PR if you feel like it.)